<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Definitions</class>
 <widget class="QWidget" name="Definitions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Definitions</string>
  </property>
  <widget class="QLabel" name="header">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>720</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Definitions&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Connect the definition to its term&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="def1_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>10</number>
   </property>
   <property name="text">
    <string>Definition 1</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="def2_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>10</number>
   </property>
   <property name="text">
    <string>Definition 2</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="def3_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>10</number>
   </property>
   <property name="text">
    <string>Definition 3</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="term1_button">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>100</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>10</number>
   </property>
   <property name="text">
    <string>Term 1</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="term2_button">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>150</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>10</number>
   </property>
   <property name="text">
    <string>Term 2</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="term3_button">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>200</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>10</number>
   </property>
   <property name="text">
    <string>Term 3</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>term1_button</zorder>
  <zorder>header</zorder>
  <zorder>def1_button</zorder>
  <zorder>def2_button</zorder>
  <zorder>def3_button</zorder>
  <zorder>term2_button</zorder>
  <zorder>term3_button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
